### PR TITLE
Fix: Use ${node_bin} for the node binary

### PR DESCRIPTION
**Pull request by rchl**
With the latest version of lsp_utils a change was introduced [1] that allows using a locally
managed node runtime instead of the system one. For that to work, the "node" command
needs to use a variable.

[1] https://github.com/sublimelsp/lsp_utils/commit/403345a0c5c15e84802c712044c630a9fb236b9d

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
```
Comment (predragnikolic) [inline tox.ini:1]: Just a question. We do not have github actions in lsp-elm repo. Is this file intended to be run manually in the terminal for now or?

**Review by predragnikolic — APPROVED** [review]

File: tox.ini (hunk 1 above)
Comment (rchl) [reply tox.ini:1]: I'm running flake8 locally through LSP so I saw issues with too long lines without it.
